modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java

```diff
@@ -149,6 +154,24 @@ private void _addNestedFields(
 		}
 	}
 
+	private DecimalFormat _getDecimalFormat(Locale locale) {
```

Comment:
There is a similar method in the class NumericDDMFormFieldUtil. Could you reuse from there?

Reply:
I updated source based on your suggest. Thank you!